### PR TITLE
Moving FAQ_purging histories

### PR DESCRIPTION
**Pull request by Nurzhamalyrys**
Hi everyone!
I am moving the  FAQ  purging histories.

### CODE REVIEW

File: faqs/galaxy/histories_purging_histories.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+1. All account Histories can be reviewed under User > Histories.
+2. Locate deleted but not permemently deleted histories and purge them.
+
+Notes:
+- Download any datasets or entire histories you want to keep first. The history will need to be in an active state. Histories that are deleted can be undeleted, but once a history is purged (aka permenently deleted) it cannot be recovered, even by an administrator.
+- Histories must be unshared before they can be deleted or purged. Unshare first as needed.
+- Histories in a deleted state are still part of your quota usage.
+- Histories must be purged (permenently deleted) to not count toward quota
```
Comment (gallardoalba) [inline faqs/galaxy/histories_purging_histories.md:16]:
```suggestion
- Histories must be purged (permanently deleted) to not count toward quota.
```

File: faqs/galaxy/histories_purging_histories.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: purging unneeded histories
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_purging_histories.md:2]:
```suggestion
title: Purging unneeded histories
```

File: faqs/galaxy/histories_purging_histories.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Purging unneeded histories
+area: histories     
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys]
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_purging_histories.md:6]:
```suggestion
contributors: [jennaj, nurzhamalyrys] 
```

File: faqs/galaxy/histories_purging_histories.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+box_type: tip        
+layout: faq        
+contributors: [nurzhamalyrys] 
+---
+
+1. All account Histories can be reviewed under **User > Histories**.
+2. Locate deleted but not permanently deleted histories and purge them.
+
+Notes:
+- Download any datasets or entire histories you want to keep first. The history will need to be in an active state. Histories that are deleted can be undeleted, but once a history is purged (aka permenently deleted) it cannot be recovered, even by an administrator.
```
Comment (beatrizserrano) [inline faqs/galaxy/histories_purging_histories.md:13]:
```suggestion
- Download any datasets or entire histories you want to keep first. The history will need to be in an active state. Histories that are deleted can be undeleted, but once a history is purged (aka permanently deleted) it cannot be recovered, even by an administrator.
```

File: faqs/galaxy/histories_purging_histories.md (hunk 3 above)
Comment (beatrizserrano) [reply faqs/galaxy/histories_purging_histories.md:6]: @Nurzhamalyrys you can commit directly in the suggestion here, no need to edit the file

**Comment by annefou** [comment]
This FAQ has also been finalized in another PR #3360 so we can close it too. Thanks a lot.

**Pull request closed without merging**